nixos/modules/services/networking/firewall.nix

```diff
@@ -177,6 +177,23 @@ let
       ) cfg.allowedUDPPortRanges
     }
 
+    # Accept packets on the extra allowed host+port combination.
+    ${concatMapStrings (allowed:
+        concatMapStrings (addressFamily:
+        concatMapStrings (protocol:
+          (if addressFamily == "IPv4" then "iptables -A nixos-fw "
+           else "ip6tables -A nixos-fw ")
```

Comment:
Oh, and could we move that part to the end of the "static" stuff (imho right above `${cfg.extraCommands}` would be the best place)?

That way reading the output of e.g. `iptables -nvL` would be easier (the default rules first since they normally don't change and then the custom stuff).